fuzz/fuzz_pcap.c

```diff
@@ -0,0 +1,115 @@
+#include <config.h>
+#include "netdissect-stdinc.h"
+#include <sys/stat.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <string.h>
+
+#include <pcap/pcap.h>
+#include "netdissect.h"
+#include "print.h"
+
+FILE * outfile = NULL;
+
+static int
+fuzz_ndo_printf(netdissect_options *ndo, const char *fmt, ...)
+{
+    va_list args;
+    int ret;
+
+    va_start(args, fmt);
+    ret = vfprintf(outfile, fmt, args);
```

Comment:
I had trouble with getting EINTR when running a long fuzz, so I switched to using `vasprintf()` (and freeing the resulting string) so that the arguments got formatted but we could discard them.  I also have an option triggered by an environment variable to leave ndo->ndo_printf alone, so that you get the output when you run a replication case.

Reply:
Interesting, I did not get this `EINTR`.
Was you file /dev/null or some regular file for logging ?
Your solution looks good.